docs/sources/setup-grafana/configure-security/configure-authentication/_index.md

```diff
@@ -2,6 +2,11 @@
 aliases:
   - ../../auth/
   - ../../auth/overview/
+cascade:
+  labels:
+    products:
```

Comment:
add enterprise

Reply:
This file is complicated. We have a cloud-specific file on OAuth2, and it's very similar, but configuration might be different (issue open for someone to do a complete OAuth audit/update). SAML is not available in cloud.

Reply:
@Eve832 Would you prefer I remove cloud for now?

Reply:
Given what @ddorman16 has said, I'm inclined to leave it in.